app/assets/javascripts/posts/editor.js

```diff
@@ -38,6 +38,12 @@ function setupMetadataEditor() {
     placeholder: 'Choose user(s) to view this post'
   });
 
+  $("#post_unjoined_author_ids").select2({
+    width: '300px',
+    minimumResultsForSearch: 20,
+    placeholder: 'Choose user(s) invited to reply to this post'
```

Comment:
"invited" → "to invite" would sound more natural?

"Choose user(s) to invite to reply to this post".